pages/api/mindmaps/index.js

```diff
@@ -68,6 +68,9 @@ const MindMapsAPI = async (req, res) => {
         return res.status(response.statusCode).json({ message })
 
       case 'PATCH':
+        if (req.body._id ===  undefined) {
```

Comment:
If we're checking, better to use _.isEmpty, to catch all types of blanks (null, empty string, undefined, 0, etc).

Reply:
I think this is perfectly fine as it is now